kubernetes/loculus/templates/loculus-database-standin.yaml

```
@@ -42,6 +42,6 @@ spec:
           value: "trust"
         {{ if not .Values.developmentDatabasePersistence }}
         - name: LOCULUS_VERSION
-          value: {{ $dockerTag }}
+          value: "commit-c6268d7"
```

To revert before merging:
```suggestion
          value: {{ $dockerTag }}
```